### ISSUE TITLE
Cannot reference another mount point

### ISSUE THREAD

**Issue by kptdobe** [adobe/helix-simulator#503]
Use case: to work on a new feature for theblog which requires some content change, I need to reference another sharepoint folder with content I can update (to not interfere with the production content).

Steps:
- clone theblog
- create a new "mytestbranch" branch locally
- change the mount point in fstab.yaml to another sharepoint folder ("testfolder")
- copy /en/topics/_taxonomy.docx document in "testfolder"
- make some changes in the document
- hlx up
- request http://localhost:3000/en/topics/_taxonomy.plain.html

-> the document served is not the one from "testfolder"

I did not identify yet where the issue is coming from exactly (if this is the branch not being pushed - mountpoint not up-to-date - or something wrong in our logic but logs are already showing a wrong branch:

```
[hlx] info: simulator proxy: try loading from github first: http://127.0.0.1:64744/raw/helix/github.com--adobe--theblog/mytestbranch/en/topics/_taxonomy.md
[hlx] info: Could not find file at http://127.0.0.1:64744/raw/helix/github.com--adobe--theblog/mytestbranch/helix-markup.yaml
[hlx] info: simulator proxy: http://127.0.0.1:64744/raw/helix/github.com--adobe--theblog/mytestbranch/en/topics/_taxonomy.md does not exist. 404
[hlx] info: simulator proxy: fetch from content proxy https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Ftopics%2F_taxonomy.md&ref=master&ignore=github
[hlx] info: simulator proxy: fetch from content proxy https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Ftopics%2F_taxonomy.md&ref=master&ignore=github: 200
```

In the last 2 logs (`fetch from content proxy`), `ref` is `master` which is wrong, should be `mytestbranch`. I tried to change that but it does not fix the issue.

cc @tripodsan @trieloff

**Comment by tripodsan** [comment]
> I tried to change that but it does not fix the issue.

where did you try to change that ?

**Comment by kptdobe** [comment]
Here: https://github.com/adobe/helix-simulator/blob/master/src/utils.js#L211-L223
But I do not understand why this code copies the original strain and not the query params. Actually, I do not understand why the original strain / content refers to master and not "mytestbranch".

**Comment by tripodsan** [comment]
well, it takes the `helix-config.yaml` from helix-pages, which has a default strain with the content repo ref `master`, I think.

the `strain.originalContent` is the one specified in the `helix-config.yaml`. the `strain.content` is the one that is overridden with the local github server. so it should probably use the `strain.content.ref` instead, to mimic the local checkout.

**Comment by tripodsan** [comment]
@kptdobe did you ever commit your changes to the testbranch ? how should the content proxy be able to load your local branch ?

**Comment by kptdobe** [comment]
The files are changed "locally", is it expected that I commit them ? If I change locally a `js` file, I do not need to commit it.

**Comment by tripodsan** [comment]
yes. but the simulator doesn't simulate the content proxy completely - i.e. it invokes the content-proxy in runtime - which can't read you local changes.

anyways, I tested it, and it doesn't work yet.... fix comming.
